source/educators/quickstarts/build_a_course.rst

```diff
@@ -78,8 +80,26 @@ Follow the steps below to build your first course. By the end, you will have a f
 .. seealso::
  :class: dropdown
 
+ :ref: `Creating a New Course` (how-to)
```

Comment:
`:ref:` cannot have a space between the colon and the backtick. Look at the rendered page: https://docsopenedxorg--795.org.readthedocs.build/en/795/educators/quickstarts/build_a_course.html